src/Components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,32 @@
+import cn from 'classnames';
+
+type Props = {
+  messageError: string;
+  setMessageError: (_: string) => void;
```

Comment:
`_` use only when you need to skip one argument and use others, this is not the case